src/udp_wrap.cc

```diff
@@ -219,6 +221,36 @@ void UDPWrap::Bind6(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UDPWrap::SetRecvBufferSize(const FunctionCallbackInfo<Value>& args) {
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK(args[0]->IsUint32());
+
+  int inputSize = args[0]->Uint32Value();
```

Comment:
`Uint32Value` without v8::Context specified is deprecated. Use `args[0]->Uint32Value(info.GetIsolate()->GetCurrentContext())`.